extras/rpms/glusterd2.spec

```diff
@@ -35,34 +36,25 @@ Preview release of the next generation GlusterFS management framework and daemon
 
 %build
 export GOPATH=$(pwd):%{gopath}
-
 mkdir -p src/%(dirname %{import_path})
 ln -s ../../../ src/%{import_path}
 
 pushd src/%{import_path}
 # Build glusterd2
-make glusterd2
-make glustercli
+%{gd2make} glusterd2
+%{gd2make} glustercli
+%{gd2make} glusterd2.toml
```

Comment:
Just a question: make a conf file, what does this do ?

Reply:
Generates glusterd2.toml with proper paths set for statedir, logdir etc.